apps/spruce/src/components/Settings/EventLog/EventDiffTable.tsx

```diff
@@ -69,11 +80,17 @@ const renderEventValue = (value: EventValue): string => {
   return JSON.stringify(value);
 };
 
-const columns: LGColumnDef<EventDiffLine>[] = [
+const columns = (
+  customKeyRenderConfig: CustomKeyRenderConfig = {},
+): LGColumnDef<EventDiffLine>[] => [
   {
     header: "Property",
     accessorKey: "key",
-    cell: ({ getValue }) => <CellText>{getValue() as string}</CellText>,
+    cell: ({ getValue }) => (
+      <CellText>
```

Comment:
This may be more helpful if it were applied on the REDACTED section instead of the variable name. The redacted text is the change that we're actually drawing attention to, not the variable name.

Reply:
That was actually what I originally implemented before I realized we had a [design](https://www.figma.com/design/EgYGXJtGBz43B6StlL46B2/EXPO-4044%3A-Project-Var-Hash-Updates?node-id=48-2624) linked haha. If you think that makes more sense I can move it.

Reply:
Ha! I do think moving it makes more sense, especially since those designs still include the variable values so maybe it wasn't super clear what the ask was.